spec/factories/application_qualification.rb

```diff
@@ -126,7 +126,9 @@
 
       trait :bachelor do
         qualification_level { 'bachelor' }
-        qualification_type { Hesa::DegreeType.where(level: :bachelor).first.name }
+        qualification_type {
+          ['Bachelor of Arts', 'Bachelor of Engineering', 'Bachelor of Science', 'Bachelor of Education'].sample
```

Comment:
The 'first' one worked  previous on all our tests because it was one of the high-level options, but that was incidental. Now there are more degrees and the first one is not one of the high level options used for our radio buttons. This sample is.